public-ui/src/app/data-browser/chart/chart.component.ts

```diff
@@ -296,6 +313,7 @@ export class ChartComponent implements OnChanges {
       // Series name for answers is the answer selected which is in stratum4
       seriesName = this.selectedResult.stratum4;
     }
+    console.log(results);
```

Comment:
Assuming this was left in from debugging, please remove

Reply:
Done.